test/jasmine/tests/axes_test.js

```diff
@@ -4376,6 +4376,33 @@ describe('Test axes', function() {
             })
             .then(done, done.fail);
         });
+        it('should respect axis title placement on relayout', function(done) {
+            function getPos(gd, sel) {
+                return d3Select(gd).select(sel).node().getBoundingClientRect();
+            }
+
+            // Tick position is < title position since 0 is at the top of the graph,
+            // rather than at the bottom. We're checking that the ticks and title don't overlap
+            function assertLayout() {
+                var titleTop = getPos(gd, '.xtitle').top;
+                var tickBottom = getPos(gd, '.xtick').bottom;
+                expect(tickBottom).toBeLessThan(titleTop + 2); // allow two pixels tolerance
+            }
+
+            var fig = require('@mocks/z-automargin-zoom.json');
+            Plotly.newPlot(gd, fig)
+
+            .then(function() {
+                assertLayout();
+            })
```

Comment:
```suggestion
            .then(assertLayout)
```
Not a big deal, just a little cleaner (here and the other `assertLayout` call)

Reply:
Good point!